fe/fe-core/src/main/java/com/starrocks/authentication/AuthenticationMgr.java

```diff
@@ -740,8 +740,6 @@ public void loadV2(DataInputStream dis) throws IOException, SRMetaBlockException
                 }
             } catch (AuthenticationException e) {
                 throw new RuntimeException(e);
-            } finally {
-                reader.close();
```

Comment:
why remove it? miss checksum checking?

Reply:
The above close has been verified, here is the second verification, this is a bug in the previous PR